packages/quill/src/core/theme.ts

```diff
@@ -27,6 +28,14 @@ class Theme {
     protected options: ThemeOptions,
   ) {}
 
+  detach() {
+    Object.values(this.modules).forEach((module) => {
+      if (module instanceof Module) {
```

Comment:
I wonder if instead a better check might be `if ('detach' in module)`. You _can_ register things with quill that aren't literally subclasses of `Module`